src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const successMessage = document.createElement('div');
+const errorMessage = document.createElement('div');
+
+successMessage.classList.add('message');
+successMessage.innerText = 'Promise was resolved!';
+errorMessage.classList.add('message', 'error-message');
+errorMessage.innerText = 'Promise was rejected!';
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', (e) => {
+    resolve();
+  });
```

Comment:
The promise `promise1` resolves when the logo is clicked, but there is no rejection condition. It's a good practice to always have a rejection path in promises for handling unexpected situations. Consider adding a rejection condition or a timeout for rejection.